vllm/attention/backends/torch_sdpa.py

```diff
@@ -158,6 +158,11 @@ def forward(
             shape = [num_tokens, num_heads * head_size]
         """
         assert kv_scale == 1.0
+        if attn_type != AttentionType.DECODER:
+            raise NotImplementedError("Encoder self-attention and " + \
+                                      "encoder/decoder cross-attention " + \
+                                      "are not implemented for " + \
+                                      "TorchSDPABackendImpl")
```

Comment:
nit: You actually don't need `\` here.

Also, it is recommended to avoid `\`: https://google.github.io/styleguide/pyguide.html#32-line-length

Reply:
ditto for other places.